### PR TITLE
chore: rename master -> main; update links

### PR DESCRIPTION
**Pull request by thomasheartman**
update links and workflows after branch renaming.

Also fixes Cake issues by using the changes from @daveleek's PR #84 (thanks for figuring that out and kudos! 😄).

### CODE REVIEW

**Comment by thomasheartman** [comment]
[Rename GitHub default branches from master to main](https://trello.com/c/Ey6olr8s/669-rename-github-default-branches-from-master-to-main)

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ The .Net client comes with implementations for the built-in activation strategie
 - ApplicationHostnameStrategy
 - FlexibleRolloutStrategy
 
-Read more about the strategies in [activation-strategy.md](https://github.com/Unleash/unleash/blob/master/docs/activation-strategies.md).
+Read more about the strategies in [activation-strategy.md](https://github.com/Unleash/unleash/blob/main/docs/activation-strategies.md).
```
Comment (chriswk) [inline README.md:121]:
```suggestion
Read more about the strategies in [activation-strategy.md](https://docs.getunleash.io/user_guide/activation_strategy).
```
Linking to the compiled version of the docs seems more logical, also makes branch name uninteresting :)

File: README.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ IUnleash unleash = new DefaultUnleash(config, s1, s2);
 
 ### Unleash context
 
-In order to use some of the common activation strategies you must provide an [unleash-context](https://github.com/Unleash/unleash/blob/master/docs/unleash-context.md).
+In order to use some of the common activation strategies you must provide an [unleash-context](https://github.com/Unleash/unleash/blob/main/docs/unleash-context.md).
```
Comment (chriswk) [inline README.md:135]:
```suggestion
In order to use some of the common activation strategies you must provide an [unleash-context](https://docs.getunleash.io/user_guide/unleash_context).
```
Same as for activation strategies

**Review by chriswk — APPROVED** [review]
Slight nit with suggesting to link to the compile docs rather than the md file :)

File: README.md (hunk 1 above)
Comment (chriswk) [reply README.md]: Particularly since the docs folder no longer contains that markdown file?

**Pull request merged**